### ISSUE TITLE
Add back original history

### ISSUE THREAD

**Issue by ZedThree** [boutproject/boutdata#21]
This was originally added as a straight dump of the then-state of the files in BOUT-dev. We could "add back" the original history. This would completely divorce any existing tags or branches from the new history, so is maybe not such a great idea.

For reference, this is how I've just done it with Zoidberg:

```
mkdir zoidberg && cd zoidberg
git init
git remote add --fetch https://github.com/bout-project/BOUT-dev.git
git cherry-pick -Xsubtree=tools/pylib \
  $(git log --reverse --pretty=format:"%h" --follow bout/next -- tools/pylib/zoidberg)
```

- `git log ...` to get all the commits that touch zoidberg files
- `cherry-pick -Xsubtree=tools/pylib` add those commits one at a time, stripping the `tools/pylib` path so the files end up under `zoidberg/`

Requires some cleaning up along the way, but generally there aren't many conflicts -- it's usually just deleting other files brought in in those commits.

If we were to do this, the general plan of attack would be:

1. create temporary branch from initial commit
2. do above method to recreate the original history
3. rebase master and any other branches on top of this

**Comment by johnomotani** [comment]
Since we've got tags linked to releases on PyPi, I'd be inclined not to do this.

Maybe a compromise would be to create a 'pre-history' branch using the git magic, containing the history up to the point boutdata/boututils split, but not merge/rebase master? The history's there in BOUT-dev anyway, but it might be a bit easier to search like this...

**Comment by ZedThree** [comment]
Yeah, that was my main worry too. I think that they would be sort of ok, they might end up still existing just not on a branch. Still, pretty wary of messing with them!

`prehistory` branch does like a good compromise though.

**Comment by dschwoerer** [comment]
It has been a few years, so probably time to close this. The history is still in the BOUT-dev repo.
